rollup.config.js

```diff
@@ -10,9 +11,10 @@ const libName = 'temporal';
 
 export default [
   {
-    input: 'lib/index.mjs',
+    input: 'lib/index.ts',
     plugins: [
-      replace({ exclude: 'node_modules/**', __debug__: !isProduction }),
+      typescript(),
+      replace({ exclude: 'node_modules/**', 'globalThis.__debug__': !isProduction, preventAssignment: true }),
```

Comment:
Yes, because this replacement happens over the emitted js, not the source ts file.

I tested this by running `NODE_ENV=production npm run build`, and grepping the output file for `_repr_`, which is one of the common properties that is only set in debug builds - it is still removed from the build with this change.

Reply:
While you're changing this, could you also fix the rollup warning with this plugin?  Here's what I see when I run `npm build` on this branch:
```
(!) Plugin replace: @rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option to `true`, as the next major version will default this option to `true`.
```

Reply:
Er, never mind the comment above-- I was on the wrong branch.  Build works fine.